docs/user-guide/steerlm.rst

```diff
@@ -108,6 +114,53 @@ Finally, preprocess the data into regression reward model training format.
       --output-file=data/merge_val_reg.jsonl
 
 
+If you are interested in following the approach from HelpSteer2 work, please do the below instead.
```

Comment:
Can you provide a bit more detail on what it means to "follow the approach"? A brief explanation of the approach and how it differs from the standard steerlm approach would be useful.

Reply:
The approach is the exactly the same as standard SteerLM approach. I changed the wording to "If you are interested in replicating Reward Modelling training in HelpSteer2, please do the below instead."